config/configauth/configauth.go

```diff
@@ -28,7 +29,7 @@ type Authentication struct {
 
 // GetServerAuthenticator attempts to select the appropriate auth.Server from the list of extensions,
 // based on the requested extension name. If an authenticator is not found, an error is returned.
-func (a Authentication) GetServerAuthenticator(extensions map[component.ID]component.Component) (auth.Server, error) {
+func (a Authentication) GetServerAuthenticator(_ context.Context, extensions map[component.ID]component.Component) (auth.Server, error) {
```

Comment:
@AkhigbeEromo since this is a breaking change it will need done over several releases. In this case, we need to deprecate `GetServerAuthenticator` and `GetClientAuthenticator` and add new `GetServerAuthenticatorContext` and `GetClientAuthenticatorContext` functions. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes

Reply:
Thank you @TylerHelmuth , will work on this right away

Reply:
Done @TylerHelmuth